Remotion/Web/Development.WebTesting.UnitTests/SetUpFixture.cs

```diff
@@ -0,0 +1,32 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests
+{
+  [SetUpFixture]
+  public class SetUpFixture
+  {
+    [OneTimeSetUp]
+    public void OneTimeSetUp ()
+    {
+      WebTestSettings.Current = WebTestSettings.CreateAppConfigBasedWebTestSettings();
```

Comment:
Unit test: wouldn't it be easy/helpful to use json-based here?

Reply:
It doesn't really matter here. We already have a `Web.config` that is setup for the tests and is used in unrelated tests. All the new tests do not rely on `WebTestSettings.Current` so a change here would not have an effect on the new tests.

Reply:
Ah, I see. we use a config file for unit testing some of the features (i.e. WebTestHelper.CreateFromConfiguration). Given we will be deprecating and then removing the xml-based config anyway in the future, wouldn't it be best to switch to either an appsettings based version or create the config in code?

Reasoning: leaving legacy code as the main version we are working with feels like we're pushing cleanup down the road when we could do it right now anyway.

It would be a good idea to one day move access to the static WebTestSettings.Current property into the ctor of WebTestConfigurationFactory so we only have a single access and then we can refector the tests to better isolate this concern.